js/src/modal.js

```diff
@@ -459,64 +459,47 @@ class Modal extends BaseComponent {
       // Note: DOMNode.style.paddingRight returns the actual value or '' if not set
       //   while $(DOMNode).css('padding-right') returns the calculated value or 0 if not set
 
-      // Adjust fixed content padding
-      SelectorEngine.find(SELECTOR_FIXED_CONTENT)
-        .forEach(element => {
-          const actualPadding = element.style.paddingRight
-          const calculatedPadding = window.getComputedStyle(element)['padding-right']
-          Manipulator.setDataAttribute(element, 'padding-right', actualPadding)
-          element.style.paddingRight = `${Number.parseFloat(calculatedPadding) + this._scrollbarWidth}px`
-        })
+      const setElement = (selector, styleProp, callback) => {
+        SelectorEngine.find(selector)
+          .forEach(element => {
+            const actualValue = element.style[styleProp]
+            const calculatedValue = window.getComputedStyle(element)[styleProp]
+            Manipulator.setDataAttribute(element, styleProp, actualValue)
+            element.style[styleProp] = callback(Number.parseFloat(calculatedValue)) + 'px'
+          })
+      }
```

Comment:
It will be a little optimized way if you move this function to a private method. Because it will no be defined again and again.

Reply:
Yeah and probably `resetElement()` too :)